### ISSUE TITLE
Error on https://pytorch.org/tutorials/intermediate/tensorboard_tutorial.html ?

### ISSUE THREAD

**Issue by motiwari** [pytorch/tutorials#1104]
In Section 6 of https://pytorch.org/tutorials/intermediate/tensorboard_tutorial.html, the following code is presented:

```
def add_pr_curve_tensorboard(class_index, test_probs, test_preds, global_step=0):
    '''
    Takes in a "class_index" from 0 to 9 and plots the corresponding
    precision-recall curve
    '''
    tensorboard_preds = test_preds == class_index
    tensorboard_probs = test_probs[:, class_index]

    writer.add_pr_curve(classes[class_index],
                        tensorboard_preds,
                        tensorboard_probs,
                        global_step=global_step)
    writer.close()
```

`writer.add_pr_curve` should take the _true_ labels, not a variable testing equality between the prediction and the class index (from `https://pytorch.org/docs/stable/tensorboard.html`), so I think the second argument, `tensorboard_preds`, is wrong:  `tensorboard_preds = test_preds == class_index` tests whether the prediction is equal to the given class_index.

I think this should be something like `tensorboard_preds = labels == class_index`, which gives you 0/1 based on the true labels and not the predicted labels

**Comment by brianjo** [comment]
Thanks for the note Mo. @orionr Could you take a look?

**Comment by orionr** [comment]
@cryptopic, @natalialunova  and @lanpa might be able to help here.

**Comment by timothylimyl** [comment]
@orionr:

As of today, issue is still present. There's a conflict between the example and documentation.

**Comment by orionr** [comment]
@cryptopic can you take a look? Thanks.

**Comment by kiersten-stokes** [comment]
I believe this was fixed by #1469 and can be closed

**Comment by sekyondaMeta** [comment]
Closing this as it was updated with a PR
